website_sale_delivery_date/static/src/js/website_sale_delivery_date.js

```
@@ -0,0 +1,89 @@
+odoo.define("website_sale_delivery_date.delivery_date", [], function () {
+    "use strict";
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)